src/libstd/keyword_docs.rs

```diff
@@ -1624,8 +1624,71 @@ mod dyn_keyword {}
 //
 /// The [Rust equivalent of a C-style union][union].
 ///
-/// The documentation for this keyword is [not yet complete]. Pull requests welcome!
+/// A `union` looks like a [`struct`] in terms of declaration, but all of its
+/// fields exist simultaneously, superimposed over one another. For instance,
```

Comment:
```suggestion
/// fields exist in the same memory, superimposed over one another. For instance,
```

"simultaneously" suggests time, whereas people tend to use metaphors based on space.